src/learner/tests/test_learner.py

```diff
@@ -15,7 +15,7 @@ def create_user(cls, feature_vector):
 
     @classmethod
     def create_document(cls):
-        return Document(url=u'', title=u'', db_key=0)
+        return Document.make_from_scratch(url=u'', title=u'', summary=None)
 
```

Comment:
I found it nicer to have None instead of u'' or [].
Could you just replace all occurrences of u'' and [] by None ? :)